setup.cfg

```diff
@@ -1,3 +1,12 @@
+[tool:pytest]
+filterwarnings =
+        ignore::sqlalchemy.exc.SADeprecationWarning
+        ignore::sqlalchemy.exc.SAWarning
```

Comment:
why ignore these?

Reply:
I think we get warnings from CKAN core that we can't do anything about. This was copied from Keitaro.